pykafka/simpleconsumer.py

```diff
@@ -194,7 +194,7 @@ def start(self):
 
         # Figure out which offset wer're starting on
         if self._reset_offset_on_start:
-            self._reset_offsets()
+            self.reset_offsets()
             # make sure the reset is saved in kafka before it can rebalance
```

Comment:
Here in `start()`, this is followed immediately by `commit_offsets`. Would it make sense to append that at the end of `reset_offsets` (so something like `if self._consumer_group is not None: commit_offsets()`) instead?

One scenario where that makes a difference is if the user calls `reset_offsets` and only then `start`s the consumer: in that case, if we didn't commit the offsets yet, `start` would overwrite them with the output of `fetch_offsets`, I think?